### ISSUE TITLE
1.5 Trillion Dollar Company

### ISSUE THREAD

**Issue by BullOnMars** [google/recaptcha#524]
How can a 1.5 trillion dollar public company come up with the most disgusting, annoying and time wasting trash software like this, this is one of the worst things that happened to the internet. Cloudflare's solution is 100 times better than this absolute abomination, the only input it requires from a user is one click.

Do us human a favor, discontinue this embarrassment and fire the guy that comes up the solution.

Sincerely 

Thank you.

**Comment by FrostKnight** [comment]
Oh I don't think its the most disgusting annoying time wasting trash software...

I think its a totally great not insane corporation idea that is totally not purposed for using every user as a sweat shop just for the sake of trolling them and making them feel really stupid... oh wait. 

:P

**Comment by BullOnMars** [comment]
Finally sold my $GOOGL stocks, due to lack of innovation and tons of failed products launch in recent decades.

Also severely lacking behind Apple, Microsoft in every perspective, including consumer hardware, software, SoC etc...

Only thing Google does now is advertising, data mining, youtube and Recapcha.

Sad.

**Comment by FrostKnight** [comment]
> Finally sold my $GOOGL stocks, due to lack of innovation and tons of failed products launch in recent decades.
> 
> Also severely lacking behind Apple, Microsoft in every perspective, including consumer hardware, software, SoC etc...
> 
> Only thing Google does now is advertising, data mining, youtube and Recaptcha.
> 
> Sad.

I disagree, they are arguably just as good as the other two.  Which is a totally not bad standard.

/s

All three of them are terrible data miners, advertisers and obsessed with proprietary software. Thus, it depends on what you speak of. Microsoft and Google are worse at hardware quality and planned obsolescence last I checked, but Apple is worse at right to repair then either.  Basically, if you buy proprietary software when you don't need to, like smart devices you don't need, tablets, smart devices like tv, streaming, etc...  yeah your just feeding the problem.

BSD/Linux are the options I choose whenever possible and I go further than that in trying to starve selfish proprietors of their ad revenue when possible.  None of that stuff is needed anywho, its just used to allow people with huge piles of money to stay prideful and happy about their conquests when deep down inside, they know they are but dust in the wind just waiting for the moment they disappear... and they hide their true feelings because that would mean showing they feel insufficient.

Data collection should be something you have to go out of your way to volunteer for, as in, ten pages of signatures + type thing.

That's not the truth though, its more like the other way around lol.

**Comment by immibis** [comment]
As they said on The Simpsons, you don't get rich by writing a lot of cheques. All trillion dollar companies are evil because the ones that weren't evil didn't get a trillion dollars.

**Comment by FrostKnight** [comment]
> As they said on The Simpsons, you don't get rich by writing a lot of cheques. All trillion dollar companies are evil because the ones that weren't evil didn't get a trillion dollars.

So another-words, their acceptance of bribery and/or other shady dealings are why they are evil.

👍
